js/offer.js

```diff
@@ -0,0 +1,46 @@
+import {offerType} from './data.js';
+import {typeFeatures, renderPhotos} from './util.js';
+
+const mapCanvas = document.querySelector('#map-canvas');
+
+const mapCanvasCard = document.querySelector('#card').content;
+
+const mapCanvasTemplate = mapCanvasCard.querySelector('.popup');
+
+const renderCards = (element) => {
+  element.forEach(({author, offer}) => {
+    const mapCanvasElement = mapCanvasTemplate.cloneNode(true);
+
+    const popupFeatures = mapCanvasElement.querySelector('.popup__features');
+
+    const popupPhotos = mapCanvasElement.querySelector('.popup__photos');
+
+    mapCanvasElement.querySelector('.popup__title').textContent = offer.title;
+
+    mapCanvasElement.querySelector('.popup__text--address').textContent = offer.address;
+
+    mapCanvasElement.querySelector('.popup__text--price').textContent = `${offer.price} ₽/ночь`;
+
+    mapCanvasElement.querySelector('.popup__type').textContent = `${offerType(offer.type)}`;
+
+    mapCanvasElement.querySelector('.popup__text--capacity').textContent = `${offer.rooms} комнаты для ${offer.guests} гостей`;
+
+    mapCanvasElement.querySelector('.popup__text--time').textContent = `Заезд после ${offer.checkin}, выезд до ${offer.checkout}`;
+
+    typeFeatures(offer.features, popupFeatures);
+
+    if (offer.description !== 0) {
```

Comment:
В поле **offer.description** мы ожидаем увидеть строку, а она всегда не будет равна числу 0, а значит условие будет выполняться всегда.

Нужно либо проверять на наличие этого поля:
```suggestion
    if (offer.description) { // если поля нет вообще или оно пустое, то условие не отработает
```

либо на длину его содержимого:
```suggestion
    if (offer.description.length) { // если длина равна нулю, то условие не отработает
```